app/src/main/java/ch/epfl/sweng/studdybuddy/GroupsRecyclerAdapter.java

```diff
@@ -62,18 +65,34 @@ public void setGroupList(List<Group> groupList) {
         return vh;
     }
 
+    @SuppressLint("DefaultLocale")
     @Override
-    public void onBindViewHolder(MyViewHolder holder, int position)
-    {
+    public void onBindViewHolder(MyViewHolder holder, int position){
         Group group = groupList.get(position);
         TextView newGroupCourseTextView = holder.groupCourseTextView;
         newGroupCourseTextView.setText(group.getCourse().getCourseName());
         TextView newGroupLanguageTextView = holder.groupLanguageTextView;
         newGroupLanguageTextView.setText(group.getLang());
         TextView newGroupParticipantInfoTextView = holder.groupParticipantInfoTextView;
         //newGroupParticipantInfoTextView.setText(("Particip: " + group.getParticipantNumber() + "/" + group.getMaxNoUsers()));
+        TextView newGroupCreationDateTextView = holder.groupCreationDateTextView;
+
+        Integer day =  group.getCreationDate().day;
```

Comment:
Maybe you can encapsulate the string_date creation in a method  String creationDate(Group group)?
It will be easier to test and codeClimate usually complains when there are more than 25 lines in a function, so we might have to do it later if we want to add something in onBindViewHolder ;)